public/app/plugins/datasource/grafana-azure-monitor-datasource/variables.ts

```diff
@@ -57,7 +75,7 @@ export class VariableSupport extends CustomVariableSupport<DataSource, AzureMoni
             return lastValueFrom(this.datasource.query(request));
         }
       } catch (err) {
-        return { data: [], error: { message: messageFromError(err) } };
+        return { data: [], error: new Error(messageFromError(err)) };
```

Comment:
unrelated change but I noticed that errors where not being propagated, this is because the data frame expects an error being of type `Error` (not sure when that changed)